src/cljs/main/broadfcui/page/groups/groups_management.cljs

```diff
@@ -34,8 +36,26 @@
    :-render-groups-table
    (fn [{:keys [this state]}]
      [:div {}
```

Comment:
Discussed/reviewed with Mohs, the div does need to be there since React requires ONE root node.